cloudwatch/deployment/src/test/java/io/quarkus/amazon/cloudwatch/deployment/CloudWatchAsyncClientTest.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.amazon.cloudwatch.deployment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+import jakarta.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.logging.Log;
+import io.quarkus.test.QuarkusUnitTest;
+import software.amazon.awssdk.services.cloudwatch.CloudWatchAsyncClient;
+import software.amazon.awssdk.services.cloudwatch.model.*;
+
+public class CloudWatchAsyncClientTest {
+
+    @Inject
+    CloudWatchAsyncClient client;
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class));
+
+    @Test
+    public void shouldBeAbleToPutMetricData() {
```

Comment:
I would prefer to have tests in the integration-tests project. It is the only place where we tests all combination of client types and native build.

Reply:
Right, I wasn't aware, I've delete these tests and move into the integration module, trying to mimic the other tests